modules/mqtt/destination/mqtt-worker.c

```diff
@@ -0,0 +1,281 @@
+/*
+ * Copyright (c) 2021 One Identity
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+
+#include "mqtt-worker.h"
+#include "mqtt-destination.h"
+#include "thread-utils.h"
+#include "apphook.h"
+
+#include <stdio.h>
+
+#define TIMEOUT     10000L
+
+static LogThreadedResult
+_publish_result_evaluation (LogThreadedDestWorker *self, gint result)
+{
+  switch(result)
+    {
+    case MQTTCLIENT_SUCCESS:
+      return LTR_SUCCESS;
+
+    case MQTTCLIENT_DISCONNECTED:
+      msg_error("Disconnected during publish!",
+                evt_tag_str("persist_name", log_pipe_get_persist_name(&self->owner->super.super.super)));
```

Comment:
You've used both `log_pipe_location_tag()` and `log_pipe_get_persist_name()` in internal messages.
Both serve the same purpose: to identify the destination that produces the given message.

I think we prefer `log_pipe_location_tag()` nowadays, we should use that consistently.

Reply:
solved by this comments: 1e529e912f895d584e3677dd968e0a6474e1c8a0 , d6be4ec11fc884f40019d80fd679058e260a67a5